### DISCUSSION TITLE
How to add nodes with an old Aeon S2 z-stick

### DISCUSSION THREAD

**Discussion by Westcott1** [zwave-js/zwave-js-ui#1902] [Q&A]
I've got zw2m running on an RPi4 using an old Aeon S2 z-stick.
It shows up correctly in the console, and seems to communicate OK
The method for adding new nodes to it is -

Unplug the stick (yes, really, it has a small built-in battery!)
Take it to the node to be included.
Put it and then the node, into include mode.
Watch its light to see successful inclusion.
Plug back in.

That all works OK, but no node appears in the zw2m console.
What am I missing?
It all worked OK with Domoticz...

**Comment by robertsLando** [comment]
You could also include your nodes without unplug the controller by just going to control panel, manage nodoes and select the inclusion mode you prefer. Try this method first and if it's not working send us some logs

**Reply by Westcott1** [reply]
Hi RobertsLando,
Yes, I had tried that originally.
Debug gives me this -

2021-10-29 10:52:44.409 DEBUG SOCKET: Event ZWAVE_API emitted to plvZnZCU_S0pkQrLAAAN
2021-10-29 10:52:44.411 INFO ZWAVE: Calling api startInclusion with args: [ 0, { forceSecurity: false }, [length]: 2 ]
2021-10-29 10:52:45.643 INFO ZWAVE: Failed to send the message after 3 attempts (ZW0202) startInclusion undefined
2021-10-29 10:53:15.638 ERROR ZWAVE: Failed to send the message after 3 attempts (ZW0202)

**Reply by robertsLando** [reply]
@Westcott1 Please enable logs top file and upload zwavejs logs file when you try to do an inclusion

**Reply by robertsLando** [reply]
cc @AlCalzone

**Reply by AlCalzone** [reply]
Definitely not by walking around with the stick
https://github.com/zwave-js/node-zwave-js/issues/1995

If you do that, the nodes will only show up after restarting and you won't be able to use S0/S2.

Like @robertsLando said, we need driver logs to figure out why the normal way doesn't work.


See [driver log](https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/generating-logs?id=driver-logs), make sure it's on loglevel `debug` or `silly` and **attach** it here as a file.

**Reply by Westcott1** [reply]
Thanks all for your replies.
After restarting zw2m, my first node added immediately.
